workflow/train_test_split.py

```diff
@@ -0,0 +1,167 @@
+from collections.abc import Iterable
+import numpy as np
+import pandas as pd
+import pytest
+
+
+def train_test_split(
+    df,
+    unique_key,
+    test_size=0.2,
+    seed=0,
```

Comment:
```suggestion
    seed=hash("ml-workflow"),
```

Reply:
Motivation ?

Reply:
0 is kinda special in that it is a unit for many operations (addition, xor etc)
so for example `hash(hash(34) ^ 0) == 34` which is kinda scary
this removes that scary